articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
+This scenario focuses on how to integrate Auth0 within a Consumer Identity and Access Management (CIAM) project. Auth0's recommendations are based on real-world customer implementation experience and by following the guidance provided, you will set up your project for success.
 
-Here you'll find tips on setting up user signup/login, enriching user profiles, obtaining consent for user data, and more. Let's get started!
+Customers using Auth0 for consumer-focused, also know as Business-to-Consumer (B2C),  projects typically share a common set of goals and objectives, and in the sections that follow, we'll focus on our experiences working with this shared set of expectations to help you deliver your solution efficiently.
 
-## User signup
+::: note
+The information provided is relevant to **all** project stakeholders. We recommend reading through this guidance in its entirety at least once, even if you've already started your journey with Auth0.
+:::
 
-One of the first things you'll want to set up is a user signup and login page. The [Universal Login page](/hosted-pages/login) with Lock gives you secure user authentication out-of-the-box. It supports single sign-on, passwordless login, and is customizable from the Dashboard. By using Universal Login you can focus more on the core value of your application instead of user signup.
+## Ways to integrate Auth0 and how to choose
 
-## Progressive profiling
+There are many different ways Auth0 can be integrated into the CIAM project architecture. Auth0's flexibility comprehensively supports many different use cases, however keep in mind that not every project requires 100% of the capabilities provided by Auth0.
 
-First name, last name, email, confirm email, username, password, confirm password, phone number, company name, country, address line 1, address line 2, city, state, zip code, date of birth, favorite color, shoe size, highest scrabble score.
+When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time. To help you with this, we've put together some [planning guidance](/architecture-scenarios/implementation/b2c/planning) that details our recommended strategies.
 
-A signup screen with 10-20 fields makes users hesitate before signing up. Many may not even sign up at all. To reduce barriers to entry, only require the minimum fields for signup and collect more information later. This is called [progressive profiling](/users/guides/implement-progressive-profiling). There are two ways you can perform progressive profiling with Auth0: the Management API and Rules.
+Auth0 also provides helpful implementation planning [checklists](/architecture-scenarios/checklists) that you can use organize your implementation tasks. There are six checklists that cover the following implementation steps:
```

Comment:
This paragraph (and the subsequent bullet list) I think would work better as the last paragraph(s) in this section  

```suggestion
We've also provides helpful [checklists](/architecture-scenarios/checklists) that you can use as part of your Software Development Life Cycle (SDLC) activities. There are six checklists that cover the following implementation steps:
```